moveit_ros/planning/planning_pipeline/src/planning_pipeline.cpp

```diff
@@ -302,17 +206,17 @@ bool planning_pipeline::PlanningPipeline::generatePlan(const planning_scene::Pla
               break;
             }
           if (!found)
-            problem = true;
+            isProblem = true;
         }
-        if (problem)
+        if (isProblem)
```

Comment:
Offtopic, but these multiple level deep if/else rat's nests are atrocious. I wouldn't touch these in the scope of this PR, just pointing out that this function would be a suitable candidate for a refactor after this PR as testing it should then be possible.